### ISSUE TITLE
decision: return trackerdict on analysis1axis function?

### ISSUE THREAD

**Issue by shirubana** [NREL/bifacial_radiance#165]
This comment has been on analysis1axis for a while: 

is it really desireable to return the trackerdict here?

Check if trackerdict is being updated after the analysis1axis with the Wm2Front and Wm2back ratio if this is not returned. It is in self already so it should?

**Comment by cdeline** [comment]
Yes.  We'll return it for now, but maybe not in a future release?
